### PR TITLE
pl.js update (v3.8.6)

### PR DESCRIPTION
**Pull request by xhg4**
Updated based on the 3.8.6 en.js and github's edit history from the previous Polish lang pull request.

There is an important issue with the Polish language in the game - the font used in the game doesn't support certain symboles like ą, ć, ę, ż and so on.

### CODE REVIEW

**Comment by mty22** [comment]
Just noting from @chonker1337 's work prior: https://github.com/mty22/krunker.io-translations/pull/24#issuecomment-824614605

**Comment by xhg4** [comment]
> > There is an important issue with the Polish language in the game - the font used in the game doesn't support certain symboles like ą, ć, ę, ż and so on.
> 
> I also noticed this:
> 
> image
> 
> Are there any alternative characters which we can use?

If I knew the name of the font or if I knew where to find it I could see if there are other characters in that font that could be used here but most likely there aren't such. The part that strikes me as rather odd is that "ó" is actually rendered properly

**Comment by xhg4** [comment]
> Just noting from @chonker1337 's work prior: https://github.com/mty22/krunker.io-translations/pull/24#issuecomment-824614605

Yep, mostly taken from this because the translation in that pull request was already reviewed (while those changes were still in place)

**Comment by mty22** [comment]
> @mty22 there is an error (not with the translation, but with the ingame display) when it comes to streaks - instead of saying "{0} jest na serii 5 zabójstw" (meaning "{0} is on a 5 kill streak") it says "{0} is jest na serri 5 zabójstw" (meaning "{0} is is on a 5 kill streak"). I've checked the translation file and here it is correct, there seems to be a problem with the game's code when it comes to this

Revert this one back to English. The grammar is hard coded into the game.

> Also, I have a question: in the market one of the names barely even fits the space it's supposed to be in ("Przedmiot broni białej" which translates roughly to "Melee skin"). All other names are translated in a similar way ("Przedmiot...") so we either cut out the "Przedmiot" part from all of them or leave it as it is

Unfortunately, nothing can be done about this at this current stage. :(
 
> Edit2: there is a bug - in the settings there are places where you input links wnd when you swap between the translations, the text in those places doesn't swap immedietaly (it's always 1 lang swap behind, let's say you swap from English to Polish to French, when you swap to French, the text in those places is still displayed in Polish)

Under normal circumstances, you would swap to a specific language and stick with it. Next reload of the game would be OK.

**Comment by xhg4** [comment]
Okay, the streak message is reverted to English
Waiting for a reviewer for those small changes

**Comment by mty22** [comment]
@xhg4 hey mate - do you know any other native Polish speakers who can review?

**Comment by xhg4** [comment]
> @xhg4 hey mate - do you know any other native Polish speakers who can review?

I do, I'll try to find someone willing to review this asap

**Comment by xhg4** [comment]
Nevermind, I can't really find anyone willing to review this

**Comment by mty22** [comment]
> Nevermind, I can't really find anyone willing to review this

Ah ok, thanks @xhg4  -- ill chase up!

**Comment by 8-10krunker** [comment]
> Nevermind, I can't really find anyone willing to review this

I could find someone to review this for you guys

**Comment by 8-10krunker** [comment]
> > Nevermind, I can't really find anyone willing to review this
> 
> Ah ok, thanks @xhg4 -- ill chase up!

I could find someone who is willing to read this

**Comment by mty22** [comment]
@8-10krunker go ahead! send them this pull request and let them know i'll compensate with KR!

**Comment by 8-10krunker** [comment]
> @8-10krunker go ahead! send them this pull request and let them know i'll compensate with KR!

do i send them the link to this page?

**Comment by mty22** [comment]
> > @8-10krunker go ahead! send them this pull request and let them know i'll compensate with KR!
> 
> do i send them the link to this page?

Absolutely!!

**Review by stdavious — CHANGES_REQUESTED** [review]
Hi, I was asked to do a review. The only misspelling I found is:
99. isntancje -> instancje
The rest is correct.

**Review by stdavious — APPROVED** [review]
All correct.

File: pl.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ generic.accuracy Precyzja
 player.waiting oczekiwanie na graczy
 player.died Zginąłeś
 player.killed Zabity przez
-player.killstreak {0} jest na serii {1} zabójstw
+player.killstreak {0} on a {1} Kill Streak
```
Comment (stdavious) [inline pl.js:64]: "{0} jest na serii {1} zabójstw" is correct

**Review by stdavious — DISMISSED** [review]
After reading the messages above, approved. The Polish version is absolutely correct in case you ever want to change back to Polish.

**Comment by xhg4** [comment]
> Hi, I was asked to do a review. The only misspelling I found is:
> 99. isntancje -> instancje
> The rest is correct. 

Oops, I'll fix that rn

**Comment by xhg4** [comment]
Fixed

@mty22 seems like this is ready to merge

**Comment by mty22** [comment]
Looks good! Will merge it through soon! Sorry, been busy IRL with zed's charity event etc.

**Comment by xhg4** [comment]
> Looks good! Will merge it through soon! Sorry, been busy IRL with zed's charity event etc.

No problem, take your time

I mean, the translation project doesn't seem to be even remotely close to being important to the game devs sooo

**Pull request merged**